### PR TITLE
fix(install): Skip the 0th arg as it is the script name

### LINKED ISSUE #656: I couldn't run ./install.sh on Mac but could run ./test.sh
## Important Details
```bash
# a1 @ 1deMac-mini in ~/projects/sentry/onpremise on git:master x [9:56:15] C:1
$ ./install.sh
Unexpected argument: ./install.sh. Use --help for usage information.

# a1 @ 1deMac-mini in ~/projects/sentry/onpremise on git:master x [9:57:06] C:1
$ ls
LICENSE                                    install.sh                                 sentry                                     symbolicator
README.md                                  nginx                                      sentry_install_log-2020-09-01_16-11-19.txt test.sh
cron                                       relay                                      sentry_install_log-2020-09-02_09-56-15.txt zookeeper
docker-compose.yml                         scripts                                    sentry_install_log-2020-09-02_09-57-06.txt

# a1 @ 1deMac-mini in ~/projects/sentry/onpremise on git:master x [9:57:15]
$ ./test.sh
Starting sentry_onpremise_postgres_1     ... done
Starting sentry_onpremise_redis_1        ... done
Starting sentry_onpremise_symbolicator_1 ... done
Starting sentry_onpremise_memcached_1    ... done
Starting sentry_onpremise_kafka_1        ... done
Starting sentry_onpremise_clickhouse_1   ... done
Starting sentry_onpremise_snuba-consumer_1              ... done
Starting sentry_onpremise_snuba-replacer_1              ... done
Starting sentry_onpremise_snuba-sessions-consumer_1     ... done
Starting sentry_onpremise_snuba-api_1                   ... done
02:00:02 [WARNING] sentry.utils.geo: settings.GEOIP_PATH_MMDB not configured.
/usr/local/lib/python2.7/site-packages/cryptography/__init__.py:39: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in a future release.
  CryptographyDeprecationWarning,
02:00:15 [INFO] sentry.plugins.github: apps-not-configured
/usr/local/lib/python2.7/site-packages/sentry/utils/services.py:94: UnsupportedBackend: The 'sentry.digests.backends.dummy.DummyBackend' backend for <class 'sentry.digests.backends.base.Backend'> is not recommended for production use.
  "for production use.".format(self._backend, self._base)
/usr/local/lib/python2.7/site-packages/sentry/utils/services.py:94: UnsupportedBackend: The 'sentry.tsdb.dummy.DummyTSDB' backend for <class 'sentry.tsdb.base.BaseTSDB'> is not recommended for production use.
  "for production use.".format(self._backend, self._base)
Traceback (most recent call last):
  File "/usr/local/bin/sentry", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/sentry/runner/__init__.py", line 166, in main
    cli(prog_name=get_prog(), obj={}, max_content_width=100)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/runner/decorators.py", line 30, in inner
    return ctx.invoke(f, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/runner/commands/createuser.py", line 81, in createuser
    if User.objects.filter(username=email).exists():
  File "/usr/local/lib/python2.7/site-packages/django/db/models/query.py", line 673, in exists
    return self.query.has_results(using=self.db)
  File "/usr/local/lib/python2.7/site-packages/django/db/models/sql/query.py", line 517, in has_results
    return compiler.has_results()
  File "/usr/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 858, in has_results
    return bool(self.execute_sql(SINGLE))
  File "/usr/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 887, in execute_sql
    cursor = self.connection.cursor()
  File "/usr/local/lib/python2.7/site-packages/django/db/backends/base/base.py", line 254, in cursor
    return self._cursor()
  File "/usr/local/lib/python2.7/site-packages/sentry/db/postgres/decorators.py", line 44, in inner
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/db/postgres/base.py", line 99, in _cursor
    return super(DatabaseWrapper, self)._cursor()
  File "/usr/local/lib/python2.7/site-packages/django/db/backends/base/base.py", line 229, in _cursor
    self.ensure_connection()
  File "/usr/local/lib/python2.7/site-packages/django/db/backends/base/base.py", line 213, in ensure_connection
    self.connect()
  File "/usr/local/lib/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/local/lib/python2.7/site-packages/django/db/backends/base/base.py", line 213, in ensure_connection
    self.connect()
  File "/usr/local/lib/python2.7/site-packages/django/db/backends/base/base.py", line 189, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/usr/local/lib/python2.7/site-packages/django/db/backends/postgresql/base.py", line 176, in get_new_connection
    connection = Database.connect(**conn_params)
  File "/usr/local/lib/python2.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
django.db.utils.OperationalError: could not connect to server: Connection refused
	Is the server running on host "127.0.0.1" and accepting
	TCP/IP connections on port 5432?

sentry_onpremise_clickhouse_1 is up-to-date
sentry_onpremise_smtp_1 is up-to-date
sentry_onpremise_redis_1 is up-to-date
sentry_onpremise_postgres_1 is up-to-date
sentry_onpremise_zookeeper_1 is up-to-date
sentry_onpremise_symbolicator_1 is up-to-date
sentry_onpremise_memcached_1 is up-to-date
sentry_onpremise_kafka_1 is up-to-date
sentry_onpremise_snuba-transactions-consumer_1 is up-to-date
sentry_onpremise_snuba-replacer_1 is up-to-date
sentry_onpremise_snuba-outcomes-consumer_1 is up-to-date
sentry_onpremise_snuba-api_1 is up-to-date
sentry_onpremise_snuba-sessions-consumer_1 is up-to-date
sentry_onpremise_snuba-consumer_1 is up-to-date
Creating sentry_onpremise_symbolicator-cleanup_1   ... done
Creating sentry_onpremise_relay_1                  ... done
Creating sentry_onpremise_cron_1                   ... done
Creating sentry_onpremise_post-process-forwarder_1 ... done
Creating sentry_onpremise_web_1                    ... done
Creating sentry_onpremise_snuba-cleanup_1          ... done
Creating sentry_onpremise_worker_1                 ... done
Creating sentry_onpremise_ingest-consumer_1        ... done
Creating sentry_onpremise_sentry-cleanup_1         ... done
Creating sentry_onpremise_nginx_1                  ... done
Waiting for Sentry to be up............................................................................................................An error occurred, caught SIGERR on line 39
Cleaning up...
Done.
```
thanks for helping me

### PR DESCRIPTION
**Pull request by BYK**
Fixes #656.

### CODE REVIEW

**Review by mattrobenolt — APPROVED** [review]

File: install.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ SENTRY_EXTRA_REQUIREMENTS='sentry/requirements.txt'
 MINIMIZE_DOWNTIME=
 
 load_options() {
-  while [[ -n "$@" ]]; do
+  while [[ -n "${@:1}" ]]; do
```
Comment (joshuarli) [inline install.sh:27]: `${@:1}` is kind of a special case of substring expansion `${parameter:offset}`, I had to look this up myself. I'd suggest this to make it more readable, your choice though.

```suggestion
shift  # throw away $0
  while [[ -n "$@" ]]; do
```

**Comment by ktosiek** [comment]
The problem seems to actually be with `getopt` - I've assumed GNU getopt, and MacOS uses a simpler BSD getopt. I'll prepare a patch removing the getopt call - there's only one short option anyway, and it should make the code simpler.

It's not a problem with "$@" - bash manual says "$@ Expands to the positional parameters, starting from one." https://www.gnu.org/software/bash/manual/bash.html#index-_0040 - which means getopt shouldn't get the $0, and shouldn't pass it to `load_options`.

**Comment by BYK** [comment]
Closing in favor of #660

**Pull request closed without merging**